packages/contractkit/src/wrappers/Accounts.ts

```diff
@@ -0,0 +1,153 @@
+import Web3 from 'web3'
+import { Address } from '../base'
+import { Accounts } from '../generated/types/Accounts'
+import {
+  BaseWrapper,
+  CeloTransactionObject,
+  proxyCall,
+  proxySend,
+  toTransactionObject,
+} from '../wrappers/BaseWrapper'
+
+/**
+ * Contract for handling deposits needed for voting.
+ */
+export class AccountsWrapper extends BaseWrapper<Accounts> {
+  /**
+   * Creates an account.
+   */
+  createAccount = proxySend(this.kit, this.contract.methods.createAccount)
+
+  /**
+   * Returns the attestation signer for the specified account.
+   * @param account The address of the account.
+   * @return The address with which the account can vote.
+   */
+  getAttestationSignerFromAccount: (account: string) => Promise<Address> = proxyCall(
+    this.contract.methods.getAttestationSignerFromAccount
+  )
+  /**
+   * Returns the vote signer for the specified account.
+   * @param account The address of the account.
+   * @return The address with which the account can vote.
+   */
+  getVoteSignerFromAccount: (account: string) => Promise<Address> = proxyCall(
```

Comment:
Also, maybe have a `getSigner(role, account)`

Reply:
I was trying to think about how to make them work without having to effectively do a bunch of if statements that match on the role in solidity (which I imagine would make this more inefficient.). Did you have any rough idea here?

Reply:
if you want to avoid the **evil** if, maybe you can use a  `Map<Role,Function>`

```ts
const getters = {
   [Role.Voter]: this.contract.method.getVoteSignerFromAccount,
   ...
}

return getters[role](account)
```

would check about if it needs the `bind(this.contract.methods)`

Reply:
Ah nvm, I thought this comment was on the solidity contract, not on the wrapper itself

Reply:
I added it for the authorize functions, but didn't feel like it was appropriate here as just using the direct methods seemed preferrable to me